app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,79 @@
+// TODO
+// 4. Update - PUT/PATCH /tasks/{id}
+// 5. DELETE /tasks/{id}
+
+package com.codesoom.assignment.controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.codesoom.assignment.model.Task;
+import com.codesoom.assignment.service.TaskService;
+import com.codesoom.assignment.service.TaskServiceImpl;
+
+@RestController
+@RequestMapping("/tasks")
+@CrossOrigin
+public class TaskController {
+
+    TaskService taskService = new TaskServiceImpl();
```

Comment:
특별한 이유가 없는 한 멤버 변수를 잘 숨겨주는 것이 좋습니다.
```suggestion
    private TaskService taskService = new TaskServiceImpl();
```

Reply:
넵 이런 부분을 신경써야할 것 같습니다...